charts/apisix/values.yaml

```diff
@@ -14,6 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+apisix:
+  # Enable or disable Apache APISIX itself
+  # Set it to flase and ingress-controller.enabled=true will deploy only ingress-controller
+  enabled: true
```

Comment:
There are so many top-level fields which are related with APISIX, I think we can put all of them under the apisix field.

Reply:
Yep. I did not modify it because I was worried about breaking changes.

Reply:
There is no place to reference this value yet